assembly/onpremises-ide-compiling-war-ide/pom.xml

```diff
@@ -422,12 +422,12 @@
                     </dependency>
                 </dependencies>
                 <configuration>
+                    <gwtSdkFirstInClasspath>true</gwtSdkFirstInClasspath>
```

Comment:
This is necessary for the correct classpath construction, we need to use  gwt sdk jars first in classpath.